packages/vue-components/src/ScrollTopButton.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <div
+    :class="['scroll-top-button', getIconSize(), 'd-print-none', {'lighten': $data.isLighten}]"
+    :style="iconStyle()"
+    aria-hidden="true"
+    @click="handleScrollTop()">
+    <slot name="icon">
```

Comment:
Three check warnings above, are they relevant/fixable?

Reply:
@yucheng11122017 any thoughts on the above? (In case you missed it)

Reply:
Sorry should be fixed in [7072690](https://github.com/MarkBind/markbind/pull/2170/commits/70726909b7f57f3bdd2b8fdd9219827f001e9210)!

It's about the indentation right

Reply:
Thanks, I think it's gone now. Weird that I was still seeing it when I commented earlier...